README.md

```diff
@@ -6,6 +6,10 @@ Open-source 2020 Google Internship project. ScanAndGo is a Spot-as-a-service app
 
 ## Development
 
+### Prerequisites
+1. [Node js v12.18.1](https://nodejs.org/en/download/) or recommended with a node version manager [nvm](https://github.com/nvm-sh/nvm)
```

Comment:
Nodejs 10.x is sufficient, we are currently developing/deploying to AppEngine with nodejs10 environment (nodejs 12 is not yet supported on AppEngine)

Reply:
done